src/components/Cards/PostCard.tsx

```diff
@@ -73,6 +75,13 @@ export function PostCard({ data }: Props) {
                         ? truncate(image.meta.prompt, { length: constants.altTruncateLength })
                         : image.name ?? undefined
                     }
+                    anim={
+                      image.type === 'video' &&
+                      videoMetadata?.duration &&
+                      videoMetadata.duration > MAX_ANIMATION_DURATION_SECONDS
+                        ? false
+                        : undefined
```

Comment:
Yeah the more I see all this repeated I'm certain, let's make a helper method:
```
const shouldAnimateByDefault(image:  { type: string, metadata?: MixedObject | null })
```

Will parse the metadata as VideoMetadata, and return true/false/undefined as you do  here.